vendor/gems/style-closet/app/assets/javascripts/style_closet.js

```diff
@@ -1 +1 @@
-//= require ./style_closet/
+//= require_tree .
```

Comment:
It changed because I did it wrong to begin with (the input PR) and I didn't catch it until this PR because the new inputs weren't able to be used until bootstrap was properly deprecated.